index.js

```diff
@@ -3,64 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var restify = require('restify')
-var error = require('../error')
-var bufferize = require('../bufferize')
-var config = require('../config')
-var log = require('../log')(config.logLevel, 'db-api')
-var DB = require('../db/mysql')(log, error)
-var pkg = require('../package.json')
+var bufferize = require('./bufferize')
+var version = require('./package.json').version
 
-var MemoryMonitor = require('../memory_monitor')
-var memoryMonitor = new MemoryMonitor()
```

Comment:
In auth-server we replaced it with a thin layer over hapi's own stats: https://github.com/mozilla/fxa-auth-server/blob/12e526cf05615400ef24410f3892dc163ec61a3c/bin/key_server.js#L11

We should consider a follow-up bug to get similar metrics back into this repo, but let's not block on it.